Tutorial.md

```diff
@@ -0,0 +1,45 @@
++Before you start the tutorial, we will assume that you have git installed onto your computer.
+-test	+If you don’t have git, then you can learn how to install it [here](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git). Here is a guide on the commands of [git also.](http://feedhenry.org/student-help-guide/topic04-git-introduction/book-git-tutorial/index.html#/Basics)
```

Comment:
I think this `-test` is a copy paste error?

Reply:
oops